### DISCUSSION TITLE
Want to hide Databases from specific users

### DISCUSSION THREAD

**Discussion by MrApollo** [dbeaver/dbeaver#14462] [General]
![image](https://user-images.githubusercontent.com/85119190/120510056-f6ff5d80-c3c0-11eb-96c5-979e62002174.png)
I want a specific user to be unable to see 2 of the databases and be able to see just testdb. I understand on a local instance i can make it hidden using the filter feature but I want to make this permanent on any machine with this user.

Any help would be great thanks.

**Comment by uslss** [comment]
@MrApollo you can found how to do it using SQL statement, e.g. [here](https://stackoverflow.com/questions/31862051/hide-mysql-database-from-unauthorised-different-user)

Also, you can grant or revoke needed privileges at the user tab. In my case chosen user(see picture attached) will have access for "2100" database only
![image](https://user-images.githubusercontent.com/31996417/121008514-c0985880-c79b-11eb-9bef-b6eda9a59cfd.png)

**Comment by MrApollo** [comment]
> @MrApollo you can found how to do it using SQL statement, e.g. [here](https://stackoverflow.com/questions/31862051/hide-mysql-database-from-unauthorised-different-user)
> 
> Also, you can grant or revoke needed privileges at the user tab. In my case chosen user(see picture attached) will have access for "2100" database only

The account that I want to be unable to see one of the databases still has the database "sys" that i do not want them to even see. This is after revoking every single permission.

<img width="169" alt="jas 2" src="https://user-images.githubusercontent.com/85119190/121019456-5b426880-c797-11eb-8f7f-ef3f7292df99.png">
<img width="260" alt="jas 1" src="https://user-images.githubusercontent.com/85119190/121019458-5bdaff00-c797-11eb-958b-58fa5f14f718.png">

I want the databases with the red lines to not be seen on any computer using the Jason user.
